scalikejdbc-interpolation/src/main/scala/scalikejdbc/QueryDSLFeature.scala

```diff
@@ -475,11 +475,12 @@ trait QueryDSLFeature { self: SQLInterpolationFeature with SQLSyntaxSupportFeatu
   case class InsertSQLBuilder(override val sql: SQLSyntax) extends SQLBuilder[UpdateOperation] {
 
     def columns(columns: SQLSyntax*): InsertSQLBuilder = this.copy(sql = sqls"${sql} (${sqls.csv(columns: _*)})")
+    @deprecated(message = "values is not type safe. use `namedValues` instead.", since = "2.4.0")
```

Comment:
If we can have a complete white list of types to accept for all existing JDBC drivers, throwing exception when surely invalid pameters are passed is fine. Is it really possible?